### PR TITLE
[7.4 Only] LPS-123907 Create and apply open button to form and table view

### PR DESCRIPTION
**Pull request by naraandrade**
Story: [LPS-111683](https://issues.liferay.com/browse/LPS-111683)

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:test:app-builder

**Comment by naraandrade** [comment]
ci:test:relevant

**Review by albinojunior — APPROVED** [review]
LGTM!

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/DataAndViewsTab.es.js [hunk 1]
```diff
@@ -76,7 +90,13 @@ const SelectFormView = (props) => {
 		},
 	};
 
-	return <SelectDropdown {...props} />;
+	return (
+		<div className="d-flex">
+			<SelectDropdown {...props} />
+
+			<OpenButton {...props.openProps} />
```
Comment (albinojunior) [inline modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/DataAndViewsTab.es.js:97]: Hey @naraandrade, `openButtonProps` sounds better than `openProps`, what do you think @gabrielpramos ?

Comment (albinojunior) [reply modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/DataAndViewsTab.es.js:97]: @naraandrade I think you can prevent passing of `openButtonProps` to `selectDropdown`, just deconstructing `{openButtonProps, ...props}` on the component API, what do you think?

**Review by albinojunior — APPROVED** [review]

**Comment by albinojunior** [comment]
ci:test:relevant

**Comment by albinojunior** [comment]
ci:test:sf

**Comment by albinojunior** [comment]
ci:forward

**Pull request closed without merging**